ConsoleApp_Task1/Class1.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace ConsoleApp_Task1
+{
+    public class Circle : Figure
+    {
+        private static double _Radius { get; set; }
```

Comment:
could you please make the property public and name it without _ according to the naming conventions?